include/zephyr/dt-bindings/pinctrl/mspm0-pinctrl.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2025 Texas Instruments
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _MSPM0_DT_BINDINGS_PINCTRL_H_
+#define _MSPM0_DT_BINDINGS_PINCTRL_H_
+
+#define MSP_PORT_INDEX_BY_NAME(x) ((x == "PORTA") ? 0 : 1)
+
+/* Creates a concatination of the correct pin function based on the CM
```

Comment:
"concatenation"

CM ?

And I guess this comment belongs to MSP_PINMUX() below, so move it there.